src/fonduer/parser/parser.py

```diff
@@ -32,10 +32,10 @@ class Parser(UDFRunner):
     def __init__(
         self,
         session,
+        parallelism=None,
         structural=True,  # structural information
         blacklist=["style", "script"],  # ignore tag types, default: style, script
         flatten=["span", "br"],  # flatten tag types, default: span, br
-        flatten_delim="",
```

Comment:
what did this param do before?

Reply:
Nothing from what I can tell. It's completely unused on `master`. Just stored for no reason.

Reply:
This was used back in v0.1.8 before we refactored the Parser when we implemented _flatten ourselves.

Reply:
👍